src/cli/cbenv_cmd.rs

```diff
@@ -0,0 +1,163 @@
+                .unwrap_or_else(|| String::from("")),
+            span,
+        );
+        results.add_string(
+            "cloud",
+            org.active_cloud().unwrap_or_else(|| String::from("")),
+            span,
+        );
+        results.add_string(
+            "project",
+            org.active_project().unwrap_or_else(|| String::from("")),
+            span,
+        );
+        if show_timeouts {
+            results.add_i64(
+                "management-timeout (ms)",
+                org.timeout().as_millis() as i64,
+                span,
+            );
+        }
+    } else {
+        match guard.active_cluster() {
+            Some(active) => {
+                results.add_string("username", active.username(), span);
+                results.add_string("cluster", guard.active(), span);
+                results.add_string(
+                    "bucket",
+                    active
+                        .active_bucket()
+                        .unwrap_or_else(|| String::from("<not set>")),
+                    span,
+                );
+                results.add_string(
+                    "scope",
+                    active.active_scope().unwrap_or_else(|| String::from("")),
+                    span,
+                );
+                results.add_string(
+                    "collection",
+                    active
+                        .active_collection()
+                        .unwrap_or_else(|| String::from("")),
+                    span,
+                );
+                if let Some(co) = active.capella_org() {
+                    results.add_string("capella-organization", co, span);
+                }
+
+                if show_timeouts {
+                    let timeouts = active.timeouts();
+                    results.add_i64(
+                        "data-timeout (ms)",
+                        timeouts.data_timeout().as_millis() as i64,
+                        span,
+                    );
+                    results.add_i64(
+                        "management-timeout (ms)",
+                        timeouts.management_timeout().as_millis() as i64,
+                        span,
+                    );
+                    results.add_i64(
+                        "analytics-timeout (ms)",
+                        timeouts.analytics_timeout().as_millis() as i64,
+                        span,
+                    );
+                    results.add_i64(
+                        "query-timeout (ms)",
+                        timeouts.query_timeout().as_millis() as i64,
+                        span,
+                    );
+                    results.add_i64(
+                        "search-timeout (ms)",
+                        timeouts.search_timeout().as_millis() as i64,
+                        span,
+                    );
+                }
+            }
+            None => {
+                results.add_string("username", String::from("<not set>"), span);
```

Comment:
similar comment here.  seems like the value should be some kind of optional and what you have here is really formatting.